src/rust/inetstack/protocols/layer4/tcp/tests/simulator.rs

```diff
@@ -502,15 +502,15 @@ impl Simulation {
             },
         };
 
-        let remote_qd: QDesc = match self.remote_qd {
-            Some((_, qd)) => qd.unwrap(),
+        let qd: QDesc = match args.qd {
```

Comment:
In the rest of the file, args.qd is referred to as `local_qd` or `remote_qd`. It would be good to stick with that naming for clarity.